cpu/stm32_common/include/candev_stm32.h

```diff
@@ -45,6 +45,26 @@ extern "C" {
 #error "CAN STM32: CPU not supported"
 #endif
 
+#if defined(CPU_FAM_STM32F1)
+#define ISR_CAN1_TX     isr_usb_hp_can1_tx
```

Comment:
now I get reconnects of the serial/term connection, does this ISR mean that CAN and USB cannot be used together?